docs/setup-instructions/setup-node-azure-marketplace.md

```diff
@@ -0,0 +1,37 @@
+# Setup Node via the Azure Marketplace
+
+**1. Go to Azure Marketplace -** Follow this link: <a href="https://portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home/searchQuery/Tributech" target="_blank">Azure Marketplace</a>.
+
+**2.Click on Tributech Dataspace Node** 
+
+Now your screen should look as follows: 
+
+![Startingscreen](img/setup-node-azure-startingscreen.png)
+
+**3. Choose a Hub -** Now you can select between the **DataSpace Node - Trial Hub** and **DataSpace Node - Custom Hub**. After selecting a Hub click on **create**.
```

Comment:
Please use a table for a better overview and provide information about public / private plans.